.circleci/images/primary/Dockerfile

```diff
@@ -29,12 +29,11 @@ RUN mv composer.phar /usr/local/bin/composer
 RUN composer global require hirak/prestissimo
 
 # Install XDebug.
-RUN pecl install xdebug-2.5.5 \
+RUN pecl install xdebug \
     && docker-php-ext-enable xdebug
 
 # Install Robo CI.
-# @TODO replace the following URL by http://robo.li/robo.phar when the Robo team fixes it.
-RUN wget https://github.com/consolidation/Robo/releases/download/1.1.5/robo.phar
+RUN wget http://robo.li/robo.phar
```

Comment:
Since this is code we are executing, we should be sure we're using an https URL for anything we install.

Reply:
done ;)